src/components/Picker/index.js

```diff
@@ -149,6 +150,13 @@ class Picker extends PureComponent {
 
     render() {
         const hasError = !_.isEmpty(this.props.errorText);
+
+        let scrollViewRef;
+        if (this.context != null
+            && this.context.scrollViewRef != null) {
+            scrollViewRef = this.context.scrollViewRef;
+        }
+
```

Comment:
```suggestion

        const scrollViewRef = this.context && this.context.scrollViewRef;

```

It looks like this could be simplified. I assume this will work and ends up with `scrollViewRef = undefined` which is fine to be passed as a prop.